docs/guides/mustache.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Mustache template with tailcall
+---
+Mustache templates are like fill-in-the-blank in forms for code. They let you put placeholders in your configurations, allowing dynamic content to be inserted. The dynamic content inserted within the curly braces ( {{ }} )
+Learn more about mustache template [here](https://mustache.github.io/)
```

Comment:
We should say that where ever you have a string literal used in a directive, it would support a mustache template.